src/lib/style/bootstrap4_backports/_base-import.scss

```diff
@@ -204,4 +204,28 @@
       border-color: map.get(SRC.$colors, 'gray-400');
     }
   }
+
+  // Tertiary button based on Synapse Design System
+  // See https://www.figma.com/file/0oPm5lLSUva8kyfVNMS6FA/Synapse-Style-%26-Component-Library?node-id=53%3A1239
+  .btn-tertiary {
+    @include button-outline-variant(
+      $color: map.get(SRC.$primary-color-palette, 500),
+      $color-hover: map.get(SRC.$primary-color-palette, 600),
+      $active-background: rgba(255, 255, 255, 0),
+      $active-border: map.get(SRC.$primary-color-palette, 700)
+    );
+    border: none;
+    font-weight: 900;
+    white-space: normal; // Since this appears more like a link, line breaks are ok
+    &:hover {
+      box-shadow: none;
+      text-decoration: underline;
+      text-underline-offset: 4px;
```

Comment:
@jay-hodgson 

Using an underline with `text-underline-offset` instead of `border-bottom` (which our anchor style uses). Not sure if you looked into this before. The trade-off is that this doesn't work in IE but does properly handle text-wrapping.

Should the anchor styling use this, or should this button use `border-bottom`?

Reply:
I didn't try using text-underline-offset, but this is an acceptable trade-off. I think this is a question for design then (on wrap, underline the group of text, or a line under all characters?).
If we change the solution, please be sure to verify other cases (like tabs UI!).